datacommons/datacommons.py

```diff
@@ -325,13 +325,14 @@ def get_observations(self,
                        max_rows=100):
     """Create a new column with values for an observation of the given property.
 
-    The existing pandas dataframe should include a column containing entity IDs
-    for a certain schema.org type. This function populates a new column with
-    property values for the entities.
+    The existing pandas dataframe should include a column containing population
+    dcids. This function populates a new column with observations of the
+    populations' measured property. A column containing geo ids of type City
+    can be used instead of population dcids.
 
```

Comment:
While you're here, probably worth finding out whether the 'City' limitation is really necessary. (It could just be that at some point only City had directly attached observations.)

Reply:
At least on dataCommons.org, looks like some weather observations are attached to cities ("Observation of temperature" https://browser.datacommons.org/kg?dcid=geoId/4261000). Assuming dataCommons.org is up to date, the 'City' qualification seems to be necessary.

Reply:
Looks like some of the EPA observations are also attached to cities.

Reply:
Oh, I meant to say, can it be *any* entity type rather than just City, like how get_populations() allows.

IOW, is the assert below still required... but that's a question to Bo or Antares.
    assert seed_col_type == 'StatisticalPopulation' or seed_col_type == 'City', (
        'Parent entity should be StatisticalPopulation' or 'City')